src/main/java/io/mosip/mimoto/util/JoseUtil.java

```diff
@@ -117,4 +143,27 @@ public ResponseWrapper<WalletBindingResponseDto> addThumbprintAndKeyId(ResponseW
 
         return responseWrapper;
     }
+
+    public String getJWT(String clientId) {
+
+        Map<String, Object> header = new HashMap<>();
+        header.put("alg", "RS256");
```

Comment:
It should be extracted as constant "RS256"

Reply:
This variable is not used anywhere else, so Its better inline rather than creating a new variable for that.

Reply:
updated